### DISCUSSION TITLE
Math.random(); funtion

### DISCUSSION THREAD

**Discussion by alexander-lopez-s** [cristobal-lopez/my-portafolio#38] [General]
I have 4 different photos. These photos are supposed to change the background of a DIV every 4 seconds. The problem is that sometimes it takes up to 8 seconds to change. Why is it? Is the `math.random()` function returning the same value twice??

Here is my code:

```
let image = document.getElementById("my-photos");
let images = [
  "./public/slide1.png",
  "./public/slide2.png",
  "./public/slide3.png",
  "./public/slide4.png",
];

setInterval(function () {
  let random = Math.floor(Math.random() * 4);
  image.src = images[random];
}, 4000);
```

@samirm00

**Comment by sammou00** [comment]
Yes, you have only 4 images, which means you have a 25% chance to get the same image. So it is normal.

**Comment by alexander-lopez-s** [comment]
@samirm00 i got it, thanks
